rules/REQUEST-932-APPLICATION-ATTACK-RCE.conf

```
@@ -398,6 +398,39 @@ SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAME
     setvar:'tx.inbound_anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
 
 
+# This rule detects command evasion via quotes(single/double).
+# For e.g.,
+#
+# 'i'fconfig
+# "e"cho whatever
+# 'i'p a
+#
+# Regular expression generated from regex-assembly/932290.ra.
+# To update the regular expression run the following shell script
+# (consult https://coreruleset.org/docs/development/regex_assembly/ for details):
+#   crs-toolchain regex update 932290
+#
+SecRule REQUEST_COOKIES|!REQUEST_COOKIES:/__utm/|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx [\"'][0-9A-Z_a-z]+[\"'][0-9A-Z_a-z]+|[0-9A-Z_a-z]+[\"'][0-9A-Z_a-z]+[\"']" \
```

```suggestion
SecRule REQUEST_COOKIES|REQUEST_COOKIES_NAMES|ARGS_NAMES|ARGS|XML:/* "@rx [\"'][0-9A-Z_a-z]+[\"'][0-9A-Z_a-z]+|[0-9A-Z_a-z]+[\"'][0-9A-Z_a-z]+[\"']" \
```
Exclusion for Google's UTM cookie have been removed as of v4.15.0